src/__snapshots__/migrateUIFolder.test.ts.snap

```diff
@@ -2128,6 +2249,18 @@ Object {
         ],
       },
     },
+    "organization_permissions": Object {
```

Comment:
It seems that I forgot to update the tests last PR.
This has happened for a lot of devs already, and the only way to make it poka-yoke would be to add a CI.
We could even test the CLI in the CI once https://github.com/activeviam/activeui-migration/pull/18 and https://github.com/activeviam/activeui-migration/pull/4 are merged.